spec/requests/tool_integration/baby_cost_calculator_spec.rb

```diff
@@ -0,0 +1,14 @@
+RSpec.describe 'Baby Cost Calculator', type: :request, features: [:baby_cost_calculator] do
```

Comment:
I don't think you need to wrap `features` in an array. The provided feature arguments get normalised along the way.